tdp/core/deployment/deployment_plan.py

```diff
@@ -1,6 +1,8 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+from sqlalchemy import null,text
```

Comment:
Why adding this import ?

Reply:
> Why adding this import ?

No more import null and text from sqlalchelemy in deployment_plan.py